pkg/scheduler/api/validation/validation.go

```diff
@@ -19,19 +19,27 @@ package validation
 import (
 	"errors"
 	"fmt"
+	"k8s.io/component-base/featuregate"
```

Comment:
These imports should be grouped properly. Otherwise it won't pass the CI. I guess you can run hack/update-gofmt.sh to correct these errors, or in your editor configure an instant `gofmt -w $FILE` action upon saving.

Reply:
Another thing is it seems `k8s.io/component-base` is a new dependency, we'd better avoid it unless it's mandatory.

Reply:
In order to use `utilfeature.DefaultFeatureGate.Enabled()` it expects a `featuregate.Feature`, which isn't currently imported anywhere. I would like to just check against what is enabled in the `--feature-gates` flag as @liggitt suggested (then I won't need this import and can just use `string`s) but I can't find how to access that flag and perhaps pass it to the componentconfig